### PR TITLE
Format everything

### PR DESCRIPTION
**Pull request by prokopschield**

### CODE REVIEW

**Review by robiot — APPROVED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-  }
-}
\ No newline at end of file
+    "files.exclude": {
+        "yarn.lock": true,
+        "yarn-error.log": true,
+        "node_modules": true,
+        "coverage": true
+    }
+}
```
Comment (lucemans) [inline .vscode/settings.json]: Revert please

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+                  node-version: '12.x'
+                  registry-url: 'https://registry.npmjs.org'
+            - run: yarn
+            - name: Build
+              run: yarn build
+            - name: Version
+              run: yarn version --new-version "${GITHUB_REF:11}" --no-git-tag-version
+            - run: yarn pub
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (lucemans) [inline .github/workflows/publish.yml]: Revert please

File: .github/workflows/pr_test.yml [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: '12.x'
+                  registry-url: 'https://registry.npmjs.org'
+            - run: yarn
+            - name: Test
```
Comment (lucemans) [inline .github/workflows/pr_test.yml]: Revert please

File: .github/workflows/pr_lint.yml [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: '12.x'
+                  registry-url: 'https://registry.npmjs.org'
+            - run: yarn
+            - name: Lint
+              run: yarn lint
```
Comment (lucemans) [inline .github/workflows/pr_lint.yml]: Revert please

File: .github/workflows/pr_build.yml [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v2
+              with:
+                  node-version: '12.x'
+                  registry-url: 'https://registry.npmjs.org'
+            - run: yarn
+            - name: Build
+              run: yarn build
```
Comment (lucemans) [inline .github/workflows/pr_build.yml]: Revert please

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -1,10 +1,9 @@
...
 name: Feature request
 about: Suggest improvements
-title: "[Feature]"
+title: '[Feature]'
 labels: enhancement
 assignees: robiot
-
 ---
 
 **Is your feature request related to a problem? Please describe.**
```
Comment (lucemans) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Revert please

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 7]
```diff
@@ -1,10 +1,9 @@
...
 name: Bug report
 about: Create a report to help us improve
-title: "[Bug]"
+title: '[Bug]'
 labels: bug
 assignees: robiot
-
 ---
 
 **Describe the bug**
```
Comment (lucemans) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Revert please

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 8]
```diff
@@ -20,10 +19,11 @@ A clear and concise description of what you expected to happen.
...
- - Typescript [e.g. v.5.4.5]
- - Version [e.g. v1.0.0]
+
+-   Node: [e.g. v16.9.1]
+-   OS: [e.g. Arch Linux]
+-   Typescript [e.g. v.5.4.5]
+-   Version [e.g. v1.0.0]
 
 **Additional context**
 Add any other context about the problem here.
```
Comment (lucemans) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Revert please

**Review by lucemans — CHANGES_REQUESTED** [review]

**Review by Antony1060 — APPROVED** [review]
LGTM

**Review by robiot — APPROVED** [review]

**Review by svemat01 — APPROVED** [review]

**Review by lucemans — APPROVED** [review]

**Pull request merged**